chia/wallet/wallet_node.py

```diff
@@ -865,9 +865,9 @@ async def add_states_from_peer(
         target_concurrent_tasks: int = 30
 
         # Ensure the list is sorted
-
-        before = len(items_input)
-        items = await self.wallet_state_manager.filter_spam(sort_coin_states(items_input))
+        unique_items = set(items_input)
+        before = len(unique_items)
```

Comment:
would it make sense to have an `INFO` log here if `len(unique_items) < len(items_input)`?

Reply:
Hm, maybe, but it would also require a test to get a green CI or a coverage exception :)